src/ruuvi_library_compress.c

```diff
@@ -0,0 +1,282 @@
+/**
+ * Copyright (c) 2014 - 2018, Nordic Semiconductor ASA
```

Comment:
Is the source file from Nordic or entirely your own? If your own, please mark copyright as owned by Ruuvi and BSD-3 Licensed, yourself as the author